website/admin/views/admin_user_devices.py

```diff
@@ -0,0 +1,101 @@
+from flask import render_template, flash, redirect, url_for
+
+import data_interface
+from admin import admin_site
+from admin.views.admin_user_forms import AddNewDeviceForm, SetThermostatTargetForm
+
+
+
+motion_triggers = [{'id': '00', 'name': 'When Motion is Detected'},
```

Comment:
While keeping these fields in our client code is not an elegant idea in general, we should at least try to store them only in 1 place if we need to keep them there. I'd suggest that you make a separate module where these hardcoded triggers and actions are stored.